### PR TITLE
I have implemented tables using entities. Each table has its own response. The tables are used to manage roles, add new user details and role access, and manage leave details.

### PR DESCRIPTION
**Pull request by Venkadesan0210**
I have implemented tables using entities. Each table has its own response. The tables are used to manage roles, add new user details and role access, and manage leave details.

### CODE REVIEW

File: LeaveManagement/LeaveManagement/src/main/java/com/example/LeaveManagement/entity/JwtResponse.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+public class JwtResponse {
+    private User user;
+    private String jwtToken;
+
+    public JwtResponse(User user, String jwtToken) {
+        this.user = user;
+        this.jwtToken = jwtToken;
+    }
+
+
```
Comment (shyamsundarv1489) [inline LeaveManagement/LeaveManagement/src/main/java/com/example/LeaveManagement/entity/JwtResponse.java]: Remove empty lines

File: LeaveManagement/LeaveManagement/src/main/java/com/example/LeaveManagement/entity/LeaveRequests.java [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+public class LeaveRequests {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long leaveRequestId;
+    @ManyToOne
+    @JoinColumn(name="employee_leave_id", nullable=false, referencedColumnName = "id")
+    private EmployeeLeave employeeLeave;
+
+
```
Comment (shyamsundarv1489) [inline LeaveManagement/LeaveManagement/src/main/java/com/example/LeaveManagement/entity/LeaveRequests.java]: Remove empty lines

File: LeaveManagement/LeaveManagement/src/main/java/com/example/LeaveManagement/entity/User.java [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    @ManyToMany(fetch = FetchType.EAGER, cascade = CascadeType.ALL)
+    @JoinTable(name = "USER_ROLE",
+            joinColumns = {
+                    @JoinColumn(name = "USER_ID")
+            },
+            inverseJoinColumns = {
+                    @JoinColumn(name = "ROLE_ID")
+            }
+    )
+    private Set<Role> role;
```
Comment (shyamsundarv1489) [inline LeaveManagement/LeaveManagement/src/main/java/com/example/LeaveManagement/entity/User.java]: Maintain the order for variables and methods

File: LeaveManagement/LeaveManagement/src/main/java/com/example/LeaveManagement/entity/JwtResponse.java (hunk 1 above)
Comment (Venkadesan0210) [reply LeaveManagement/LeaveManagement/src/main/java/com/example/LeaveManagement/entity/JwtResponse.java]: Addressed the above comment.

File: LeaveManagement/LeaveManagement/src/main/java/com/example/LeaveManagement/entity/LeaveRequests.java (hunk 2 above)
Comment (Venkadesan0210) [reply LeaveManagement/LeaveManagement/src/main/java/com/example/LeaveManagement/entity/LeaveRequests.java]: Addressed the above comment.

File: LeaveManagement/LeaveManagement/src/main/java/com/example/LeaveManagement/entity/User.java (hunk 3 above)
Comment (Venkadesan0210) [reply LeaveManagement/LeaveManagement/src/main/java/com/example/LeaveManagement/entity/User.java]: Addressed the above comment.

File: LeaveManagement/LeaveManagement/src/main/java/com/example/LeaveManagement/entity/JwtResponse.java (hunk 1 above)
Comment (shyamsundarv1489) [reply LeaveManagement/LeaveManagement/src/main/java/com/example/LeaveManagement/entity/JwtResponse.java]: Looks good

File: LeaveManagement/LeaveManagement/src/main/java/com/example/LeaveManagement/entity/LeaveRequests.java (hunk 2 above)
Comment (shyamsundarv1489) [reply LeaveManagement/LeaveManagement/src/main/java/com/example/LeaveManagement/entity/LeaveRequests.java]: Looks good

File: LeaveManagement/LeaveManagement/src/main/java/com/example/LeaveManagement/entity/User.java (hunk 3 above)
Comment (shyamsundarv1489) [reply LeaveManagement/LeaveManagement/src/main/java/com/example/LeaveManagement/entity/User.java]: Looks good

**Pull request merged**